### PR TITLE
[WIP] fix compilation warning

### PR DESCRIPTION
**Pull request by xXx-caillou-xXx**
i'm waiting for https://github.com/nbs-system/snuffleupagus/pull/221 to be merged to fix the last warning.

### CODE REVIEW

File: src/sp_sloppy.c [hunk 1]
```diff
@@ -41,10 +41,11 @@ static void array_handler(INTERNAL_FUNCTION_PARAMETERS,
     void (*orig_handler)(INTERNAL_FUNCTION_PARAMETERS)) {
   void (*handler)(INTERNAL_FUNCTION_PARAMETERS);
   zval func_name;
-  zval params[3] = {{{0}}};
+  zval params[3];
```
Comment (jvoisin) [inline src/sp_sloppy.c]: Can't we leave something like `zaval params[3] = {0}`?

Comment (xXx-caillou-xXx) [reply src/sp_sloppy.c]: it's either memset or `zval params[3] = {{{0},{{0, 0, 0, 0}}, {0}}};`, i find memset a bit more readable

File: src/sp_session.c [hunk 2]
```diff
@@ -13,7 +13,10 @@ ZEND_TSRMLS_CACHE_EXTERN();
 #define SESSION_G(v) (ps_globals.v)
 #endif
 
+#ifndef ZTF
```
Comment (jvoisin) [inline src/sp_session.c]: ZTF?

File: src/sp_session.c [hunk 3]
```diff
@@ -13,7 +13,10 @@ ZEND_TSRMLS_CACHE_EXTERN();
 #define SESSION_G(v) (ps_globals.v)
 #endif
 
+#ifndef ZTS
```
Comment (jvoisin) [inline src/sp_session.c]: Why not move this a bit upward, in the `#else`?

File: src/sp_sloppy.c (hunk 1 above)
Comment (jvoisin) [reply src/sp_sloppy.c]: Fair enough

**Pull request merged**